src/assets/scripts/client/InputController.js

```diff
@@ -778,29 +778,29 @@ export default class InputController {
      * @method processAircraftCommand
      */
     processAircraftCommand() {
-        let aircraftCommandParser;
         // this could use $commandInput.val() as an alternative
         const userCommand = this.input.command.trim().toLowerCase();
 
         // Using try/catch here very much on purpose. the `CommandParser` will throw when it encounters any kind
         // of error; invalid length, validation, parse, etc. Here we catch those errors, log them to the screen
         // and then throw them all at once
+        let cmd;
         try {
-            aircraftCommandParser = new CommandParser(userCommand);
+            const parser = new CommandParser(userCommand);
+            cmd = parser.parse();
         } catch (error) {
             UiController.ui_log('Command not understood', true);
-
             throw error;
         }
-
-        if (aircraftCommandParser.command !== PARSED_COMMAND_NAME.TRANSMIT) {
-            return this.processSystemCommand(aircraftCommandParser);
+        const parsedCommand = cmd;
```

Comment:
Why are we reassigning here?  Could we continue to use `cmd`?

Reply:
the reason is that I want the parsed command to be `const`, but I cannot do it in the try block as it is then only scoped the block. To make it `const`, I need to trick and use a reassign here.